src/IECoreHoudini/SOP_SceneCacheSource.cpp

```diff
@@ -421,6 +449,35 @@ void SOP_SceneCacheSource::loadObjects( const IECoreScene::SceneInterface *scene
 	}
 }
 
+void SOP_SceneCacheSource::invisibleLocations( const IECoreScene::SceneInterface *scene, bool &foundInvisible, double time )
+{
+	if ( evalInt( pVisibilityFilter.getToken(), 0, 0 ) )
+	{
+		if ( scene->hasAttribute( IECoreScene::SceneInterface::visibilityName ) )
+		{
+			IECore::ConstBoolDataPtr visibility = IECore::runTimeCast< const IECore::BoolData >( scene->readAttribute( IECoreScene::SceneInterface::visibilityName, time ) );
+			if ( !visibility->readable() )
+			{
+				foundInvisible = true;
+				return;
+			}
+		}
+	}
+
+	SceneInterface::NameList children;
+	scene->childNames( children );
+	std::sort( children.begin(), children.end(), InternedStringSort() );
```

Comment:
Why sort?

Reply:
good point too, I think it was from the code I copy but we don't need to sort here, removed in place ( same commit )